### PR TITLE
Fixed multiple body error

### PR DESCRIPTION
**Pull request by zbusia**
Sometimes document can contain more than one body element
(e.g. ForeignObject in SVG). In such case dragElm would be added
many times.

### CODE REVIEW

File: dist/angular-ui-tree.js [hunk 1]
```diff
@@ -796,6 +796,7 @@
             var dragStarted = false;
             var dragTimer;
             var body = document.body,
```
Comment (Voles) [inline dist/angular-ui-tree.js]: Is this variable still needed if you use `$body`?

**Comment by Voles** [comment]
Cool! :+1:  I've added one question, could you explain that one? Thanks!

File: dist/angular-ui-tree.js (hunk 1 above)
Comment (zbusia) [inline dist/angular-ui-tree.js]: Sometimes body.getAttribute is used in the code; I didn't clean up all to use consistently angular elements only.

**Comment by Voles** [comment]
@zbusia sorry for the late reply. Can you bring your PR up-to-date with the current release?

**Comment by Voles** [comment]
@zbusia any updates on this? Would be nice to see this fix in an upcoming release! :smiley:

**Pull request closed without merging**